layouts/shortcodes/add_collection_form.html

```diff
@@ -0,0 +1,965 @@
+          aria-expanded="false" aria-controls="collapseContact">
+          <h5 class="mb-0">
+            Contact information* <i class="bi bi-caret-down-fill"></i>
+          </h5>
+        </a>
+      </div>
+      <div id="collapseContact" class="collapse" role="tabpanel" aria-labelledby="Contact" data-bs-parent="#accordionForm">
+        <div class="card-body">
+          <h5>Collection head</h5>
+          <div class="row my-3">
+            <div class="form-group col-md-6">
+              <label for="head_firstname"><b>First name of the collection head:</b></label>
+              <input type="text" class="form-control" name="head_firstname" id="head_firstname">
+              <small class="form-text">First name of a person in charge of the collection, typically
+                PI.</small>
+            </div>
+            <div class="form-group  col-md-6">
+              <label for="head_lastname"><b>Last name of the collection head:</b></label>
+              <input type="text" class="form-control" name="head_lastname" id="head_lastname">
+              <small class="form-text">Last name of a person in charge of the collection, typically
+                PI.</small>
+            </div>
+          </div>
+          <div class="row mb-3">
+            <div class="form-group col-md-6">
+              <label for="head_role"><b>Official role of the collection head:</b></label>
+              <input type="text" class="form-control" name="head_role" id="head_role">
+              <small class="form-text">Official role of the person in charge of the collection, typically
+                PI.</small>
+            </div>
+          </div>
+          <h5>Collection contact person</h5>
+          <div class="row my-3">
+            <div class="form-group col-md-6">
+              <label for="contact_first_name"><b>First name of the contact person for enquiries:*</b></label>
+              <input type="text" class="form-control" name="contact_first_name" id="contact_first_name" required>
+              <small class="form-text">First name of the person who handles enquiries and questions regarding
+                this collection.</small>
+            </div>
+            <div class="form-group col-md-6">
+              <label for="contact_last_name"><b>Last name of the contact person for enquiries:*</b></label>
+              <input type="text" class="form-control" name="contact_last_name" id="contact_last_name" required>
+              <small class="form-text">First name of the person who handles enquiries and questions regarding
+                this collection.</small>
+            </div>
+          </div>
+          <div class="row mb-3">
+            <div class="form-group col-md-6">
+              <label for="contact_email"><b>E-mail of the contact person for enquiries:*</b></label>
+              <input type="contact_email" class="form-control" name="contact_email" id="contact_email" required>
```

Comment:
Change the `type` to `text` here (instead of `contact_email`, it should have been a typo). `pa11y` is raising an error because of that.